internal/template.go

```diff
@@ -108,6 +115,14 @@ func templateLib(workingDirectory string, templating util.Templating) error {
 	libDir, err = createDirectory(workingDirectory, "lib", true)
 	if err != nil {
 		if strings.Contains(err.Error(), "Directory already exists at") {
+			// We have added a new testing helper so we should check for that first
+			testingDir := path.Join(path.Join(workingDirectory, "lib"), "testing")
```

Comment:
Adding the new `lib/testing/tfplan.rego` file via templating to an existing templated folder
- when `lib/` exists from a previously templated rule
![Screenshot 2021-12-03 at 11 19 31](https://user-images.githubusercontent.com/81559517/144601040-8a27f9a2-d81d-4228-9a0a-b3ede624a741.png)
- when `lib/` exists but we have already added `tfplan.rego` in a previous step
![Screenshot 2021-12-03 at 11 23 36](https://user-images.githubusercontent.com/81559517/144601041-95589960-120f-40fd-b73e-3e9ef849f17b.png)